include/vcpkg/base/stringview.h

```diff
@@ -21,6 +22,7 @@ namespace vcpkg
         {
         }
 
+        StringView(const char* ptr) noexcept : m_ptr(ptr), m_size(strlen(ptr)) { }
```

Comment:
I do not like this constructor being non-explicit - it's too easy to cause UB when `ptr = null`.

Reply:
If it's explicit it defeats the entire purpose of string_view though, which is to work with literals.

(And on every implementation we care about this will immediately and deterministically crash)

Reply:
It also should be noted that `const std::string&` triggers the same UB you're worried about here :)

Reply:
Hmm, okay.

Reply:
`StringView(nullptr_t) = delete;`?